compiler/testData/codegen/box/callableReference/property/kt6870_privatePropertyReference.kt

```diff
@@ -1,8 +1,9 @@
-// IGNORE_BACKEND: JVM_IR
 // IGNORE_BACKEND: JS_IR
 // TODO: muted automatically, investigate should it be ran for JS or not
 // IGNORE_BACKEND: JS, NATIVE
 
+// FULL_JDK
```

Comment:
Why FULL_JDK directive is required? Current backend works withoit it

Reply:
Thread has a method annotated with sun.reflect.CallerSensitive, which cannot be resolved without FULL_JDK, breaking fake override generation in psi2ir. The non-IR backend works because it doesn't actually use the unresolved descriptor for anything.

Reply:
@gogabr Could you take a look? Could we add some kind of lazyness here?

Reply:
I think we should rather ignore unresolved annotation descriptors in psi2ir completely, something like this: https://github.com/JetBrains/kotlin/commit/888f7dda968b7b7debcc40af4310dc8312114d0f#diff-625389343bc48b023a77d0639f1b73b4R122. Please check if my change helps this test.

In any case, I think we shouldn't add or remove dependencies in existing codegen tests because it can mask a real problem (exactly as happened here) which will be hard to discover afterwards

Reply:
I've pushed https://github.com/JetBrains/kotlin/commit/385366384c29ac776bbfea55f7d825c3021aacd7, so `FULL_JDK` is no longer needed in this test